alignak/daemons/brokerdaemon.py

```diff
@@ -420,17 +423,23 @@ def get_new_broks(self, i_type='scheduler'):
                 logger.exception(exp)
                 sys.exit(1)
 
-    def get_retention_data(self):
+    def get_retention_data(self):  # pragma: no cover, useful?
         """Get all broks
 
+        TODO: using retention in the arbiter is dangerous and
```

Comment:
It's the retention in the broker, not in the arbiter. In the broker it can make sense